### DISCUSSION TITLE
Program schedule

### DISCUSSION THREAD

**Discussion by abanobboles** [AzuraCast/AzuraCast#3703] [General]
I tested AzuraCast and I like it a lot, it's probably the most stable way to start a web radio.

But we will start a church radio and it should work continuously without problems and without changing the system in the middle.

Now the interface with which you can set up a program for two weeks before is not easy. 
I see no way to really schedule a program. 
Do I understand something wrong (I know you can schedule a playlist) or is there a plug in with which this is easier.

Would need a solution with which people who are not IT experts can really schedule like in a calendar program.

Really looking forward to solutions or feedback from your side.

I hope my question is correct here, thank you very much.

**Comment by ItsMitchh** [comment]
Hi @abanobboles 

Thanks for posting a topic, I'll get back to you with more information on this as it's a somewhat complex topic.

**Reply by abanobboles** [reply]
Hello @SC2Mitch 

Thank you very much, that would be really great.
Thanks!

**Reply by abanobboles** [reply]
Hello @SC2Mitch 

I don't really want to stress you out and I really appreciate your message.

However, I need to make a decision soon on how we will proceed.

I think he also described the same problem here.
https://github.com/AzuraCast/AzuraCast/discussions/3676

Sorry if i disturb you.

Thank you very much.
